joomla/joomla-cms
# Updater follows Location headers for valid archive filename

Pull Request for Issue #12816 

### Summary of Changes

Changed method "download()" of class "JoomlaupdateModelDefault", found in administrator/components/com_joomlaupdate/models/default.php.
The method does now fetch the HTTP headers of the update download location `$updateInfo['object']->downloadurl->_data` until the actual file (on AWS) is reached.
It further removes the path and the query string from the URL, resulting in the actual filename of the update package, which should work fine for all platforms.

### Testing Instructions

Some people should check whether I messed up something for some specific platform setup.

### Documentation Changes Required

none
